src/components/dashboard/layout.tsx

```diff
@@ -33,7 +33,7 @@ export const DashboardLayout = (props: DashboardLayoutProps) => {
       <Flex>
         <chakra.div
           h="100vh"
-          w="17.25em"
+          w="276px"
```

Comment:
May I know why you think we should use `px` instead of `em`?

Reply:
just for consistency.

Reply:
@Emmanueldmlr We should either use `em` or tokens defined in the theme. px is discouraged because it's a static measurement, while `em` is relative

Reply:
okay then. I will update it then.